tests/Tests/Writers/LaTeX.hs

```diff
@@ -65,7 +65,7 @@ tests = [ testGroup "code blocks"
             "\\begin{description}\n\\item[foo] ~ \n\\subsection{bar}\n\nbaz\n\\end{description}"
           , "containing image" =:
             header 1 (image "imgs/foo.jpg" "" (text "Alt text")) =?>
-            "\\section{\\texorpdfstring{\\protect\\includegraphics{imgs/foo.jpg}}{Alt text}}"
+            "\\section[]{\\texorpdfstring{\\protect\\includegraphics{imgs/foo.jpg}}{Alt text}}"
```

Comment:
It's not ideal getting the empty `[]` here.  I assume that's because you're changing things into `Str ""` rather than just nothing.  Why not make `removeInvalidInline` a function `Inline -> [Inline]`?